content/administration/install/cdn.rst

```diff
@@ -4,77 +4,98 @@ Setting up a Content Delivery Network (CDN)
 
 .. _reference/cdn/keycdn:
 
-Deploying with KeyCDN_
-======================
+Deploying with KeyCDN
+=====================
 
-.. sectionauthor:: Fabien Meghazi
+.. sectionauthor:: Fabien Meghazi / Timothy Kukulka
 
-This document will guide you through the setup of a KeyCDN_ account with your
-Odoo powered website.
+A :abbr:`CDN (Content Delivery Network)` or Content Distribution Network, is a geographically
+distributed network of servers that provides high speed internet content. The
+:abbr:`CDN (Content Delivery Network)` provides quick, quality content for content heavy websites.
+
+This document will guide you through the setup of a KeyCDN_ account with an Odoo powered website.
 
 Step 1: Create a pull zone in the KeyCDN dashboard
 --------------------------------------------------
 
-.. image:: cdn/keycdn_create_a_pull_zone.png
-   :class: img-fluid
+On the KeyCDN dashboard, navigate to :guilabel:`Zones` menu item on the left. Name the
+:guilabel:`Zone Name`. This name will appear in the :abbr:`URL (Uniform Resource Locator)`. The
+:guilabel:`Zone Status` can be toggled back and forth from :guilabel:`Active` to
+:guilabel:`Inactive`. For this process the :guilabel:`Zone Type` should be :guilabel:`Pull`. Under
+:guilabel:`Pull Settings` enter the :guilabel:`Origin URL`. This should be the full database
+:abbr:`URL (Uniform Resource Locator)`.
 
-When creating the zone, enable the CORS option in the
-:guilabel:`advanced features` submenu. (more on that later)
+.. example::
+   Use ``https://yourdatabase.odoo.com``. Replace *yourdatabase* with the name of the database or the
+   custom URL.
 
-.. image:: cdn/keycdn_enable_CORS.png
-   :class: img-fluid
+.. image:: cdn/keycdn-zone.png
+   :align: center
+   :alt: KeyCDN's Zone configuration page.
 
-Once done, you'll have to wait a bit while KeyCDN_ is crawling your website.
+Under :guilabel:`General Settings` click the :guilabel:`Show all settings` button to expand the
```

Comment:
I no longer have access to the dashboard to see, as it was only a two-week trial. I wasn't referring to the screenshot, but rather, it was a screen direction; the option is at the bottom. It isn't a huge deal because by default, this is already enabled.

Reply:
Yes the General settings are on the Zone page. It is quite a long page.